GLideN64/src/Graphics/OpenGLContext/GLFunctions.cpp

```diff
@@ -342,7 +342,9 @@ extern "C" void initGLFunctions()
 	ASSIGN_PROC_ADR(PFNGLDELETEFRAMEBUFFERSPROC, DeleteFramebuffers);
 	ASSIGN_PROC_ADR(PFNGLGENFRAMEBUFFERSPROC, GenFramebuffers);
 	ASSIGN_PROC_ADR(PFNGLFRAMEBUFFERTEXTURE2DPROC, FramebufferTexture2D);
+#ifndef __APPLE__
```

Comment:
Theres a type conflict and we fixed it on the gdb pr, which seemed to work.
need to check where its used and if its a downside, otherwise i'd suggest ifdef the prototype.
Theres def a missmatch with some headers on apples side & khronos

Reply:
Oh, that's indeed way better.
I updated this PR with what ethan did on the types.
I confirm that it's compiling fine now.